index.jsx

```diff
@@ -1,13 +1,20 @@
 require("./node_modules/materialize-css/dist/css/materialize.min.css")
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Product from './Product';
+import ProductList from './ProductList';
+import mockData from './data/mockDataProducts.json';
```

Comment:
I think the mock data would be better hard coded here inline here so we don't have to commit a temporary json file.